MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m

```diff
@@ -709,15 +711,15 @@ + (MSBinaryImageType)imageTypeForImagePath:(NSString *)imagePath processPath:(NS
     NSString *appBundleContentsPath = [standardizedImagePath substringToIndex:appRange.location + 5];
 
     if ([standardizedImagePath isEqual:processPath] ||
-        // Fix issue with iOS 8 `stringByStandardizingPath` removing leading
-        // `/private` path (when not running in the debugger or simulator only)
-        [imagePath hasPrefix:processPath]) {
+            // Fix issue with iOS 8 `stringByStandardizingPath` removing leading
+                    // `/private` path (when not running in the debugger or simulator only)
+                    [imagePath hasPrefix:processPath]) {
       imageType = MSBinaryImageTypeAppBinary;
     } else if ([standardizedImagePath hasPrefix:appBundleContentsPath] ||
-               // Fix issue with iOS 8 `stringByStandardizingPath` removing
-               // leading `/private` path (when not running in the debugger or
-               // simulator only)
-               [imagePath hasPrefix:appBundleContentsPath]) {
+            // Fix issue with iOS 8 `stringByStandardizingPath` removing
```

Comment:
This one is weird too, I am wondering what's causing this. Maybe try to move the comments out of the condition.